mipidsi/src/lib.rs

```diff
@@ -255,6 +257,14 @@ where
             .write_command(dcs::SetTearingEffect(tearing_effect))
     }
 
+    ///
+    /// Returns `true` if display is currently set to sleep.
+    /// NOTE: value before call to `init` is considered undefined.
```

Comment:
Isn't it impossible to get an instance of `Display` without calling `Builder::init`?

Reply:
true that, removed